deployment/terraform/create_utils.go

```diff
@@ -35,3 +36,97 @@ func (t *Terraform) setupPrometheusNodeExporter(sshClient *ssh.Client) error {
 
 	return nil
 }
+
+// modifyRXSize modifies the size of the receiving buffer of the network card
+// to the maximum permitted by the hardware.
+func modifyRXSize(sshc *ssh.Client) error {
+	rxSizes, err := getEthtoolRxSizes(sshc)
+	if err != nil {
+		return fmt.Errorf("error getting the RX sizes from the ethtool: %w", err)
+	}
+
+	oldRxSize := rxSizes.actualRX
+
+	// Modify RX queue size to the maximum permitted
+	incRxSizeCmd := fmt.Sprintf("sudo ethtool -G $(ip route show to default | awk '{print $5}') rx %d", rxSizes.maxRX)
+	cmd := fmt.Sprintf("%s && sudo sysctl -p && sudo systemctl restart nginx", incRxSizeCmd)
+	if out, err := sshc.RunCommand(cmd); err != nil {
+		return fmt.Errorf("error running ssh command; output: %q; cmd: %q; err: %w", string(out), cmd, err)
+	}
```

Comment:
Since we implemented a `get` util, perhaps I would have expected a `set` one as well.

Reply:
I'd thought about that, but it would add a new layer of indirection for just 6 lines, without the benefit of testing (this would need an E2E test) nor de-duplication (this is the only place where we set this value), which were the main reasons for implementing the `get` util.